MonoGame.Framework/Audio/Xact/Cue.cs

```diff
@@ -73,13 +74,16 @@ public bool IsStopping
             }
         }
 
+        /// <summary>Returns whether the cue is preparing to play.</summary>
         public bool IsPreparing 
         {
             get { return false; }
         }
 
+        /// <summary>Returns whether the cue is prepared to play.</summary>
```

Comment:
With members like this one it would be good to document the WHEN a cue goes into the preparing state.

Reply:
This would certainly be a good addition, although it should be the audio documentation which goes into a deeper dive regarding the lifecycle of the api.